vars/mavenTemplate.groovy

```diff
@@ -9,7 +9,7 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
-    def mavenImage = parameters.get('mavenImage', 'fabric8/maven-builder:vd81dedb')
+    def mavenImage = parameters.get('mavenImage', 'fabric8/maven-builder:SNAPSHOT-PR-33-1')
```

Comment:
why do you use the tagged snapshot here instead of revision?

Reply:
@chmouel I'm using this PR to get a build going. I'll use revisions before the final revision of the PR.